Loading data...
Using  cuda
Training...
Process Process-2:
Traceback (most recent call last):
  File "/usr/common/software/pytorch/v1.2.0-gpu/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/common/software/pytorch/v1.2.0-gpu/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/global/homes/d/danieltm/.local/cori/pytorchv1.2.0-gpu/lib/python3.6/site-packages/wandb/wandb_agent.py", line 63, in _start
    function()
  File "sweeps/edge_sweep.py", line 210, in train
    val_acc, val_loss = edge_validate(model, val_loader, 12, device)
  File "sweeps/edge_sweep.py", line 33, in edge_validate
    edge_pred = model(data)
  File "/usr/common/software/pytorch/v1.2.0-gpu/lib/python3.6/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "sweeps/edge_sweep.py", line 158, in forward
    x = self.node_network(x, e, inputs.edge_index)
  File "/usr/common/software/pytorch/v1.2.0-gpu/lib/python3.6/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "sweeps/edge_sweep.py", line 122, in forward
    node_inputs = torch.cat(n_hop_front(x, e, edge_index, self.hops) + [x] + n_hop_back(x, e, edge_index, self.hops), dim=-1)
  File "sweeps/edge_sweep.py", line 76, in n_hop_back
    running_hop = n_hop_back(x, e, edge_index, (n-1))
  File "sweeps/edge_sweep.py", line 76, in n_hop_back
    running_hop = n_hop_back(x, e, edge_index, (n-1))
  File "sweeps/edge_sweep.py", line 74, in n_hop_back
    return [scatter_add(e[:,None]*x[end], start, dim=0, dim_size=x.shape[0])]
RuntimeError: CUDA out of memory. Tried to allocate 30.00 MiB (GPU 0; 15.78 GiB total capacity; 14.31 GiB already allocated; 11.44 MiB free; 692.48 MiB cached)
